.travis.yml

```diff
@@ -8,15 +8,36 @@ sudo: false
 addons:
   apt:
     packages:
+    - bc
+    - doxygen
+    - expect
+    - gettext
+    - libncurses5-dev
     - tree
-    - fish
+
+env:
+  global:
+    - PATH="$HOME/fish/bin:$PATH"
+    - OMF_REPO_URI="https://github.com/$TRAVIS_REPO_SLUG"
+    - OMF_REPO_BRANCH="$TRAVIS_BRANCH"
+
+before_install:
+  - tests/install_fish.sh
```

Comment:
I also don't like the directory `tests` containing scripts that aren't tests. Perhaps we could use a `tools` directory in the project root. What do you think, @bpinto?